src/test/ui/const-generics/issue-93647.stderr

```diff
@@ -1,12 +1,17 @@
-error[E0015]: cannot call non-const closure in constants
+error[E0277]: the trait bound `[closure@$DIR/issue-93647.rs:2:6: 2:8]: Fn<()>` is not satisfied
```

Comment:
This diagnostic is somewhat surprising. Why isn't it showing the `~const` when the `help` shows it?

Reply:
This is because `const_trait_impl` is not enabled.

Reply:
I guess... still the main message is very confusing. We should probably figure out how to make it obvious to someone not in the know about the latest unstable const features